fossilize_external_replayer.hpp

```diff
@@ -51,6 +51,13 @@ class ExternalReplayer
 		// Maps to --device-index.
 		unsigned device_index;
 
+		// Carve out a range of which pipelines to replay.
+		// Used for multi-process replays where each process gets its own slice to churn through.
```

Comment:
For cross-ABI concerns, this struct needs to be a POD, without an implicit constructor.. Consider adding a bool to flag if these fields should be used.